lib/robots/dor_repo/goobi/goobi_notify.rb

```diff
@@ -15,6 +15,9 @@ def initialize
         # @param [String] druid -- the Druid identifier for the object to process
         def perform_work
           object_client.notify_goobi
+        rescue Dor::Services::Client::BadRequestError => e
+          # NOTE: We don't know why this error is occurring, but the condition seems to go away momentarily, so retry automatically.
+          retry if e.message.match?(/Process template Example_Workflow does not exist/)
         end
```

Comment:
While this is an interesting idea,  if this error occurs, the robot has no idea why (missing the process tag?  typo in the process tag?  totally bogus and can be quickly retried?)  For the first two cases, a retry will not help, and while we can detect the "missing a tag' case, but not the other case.  This would lead to an infinite retry loop.

Reply:
Yes, it would.